Misc/NEWS.d/next/Core and Builtins/2023-03-07-16-56-28.gh-issue-102502.gTXrcD.rst

```diff
@@ -0,0 +1 @@
+Fix regression in semantics of normalisation in ``PyErr_SetObject``.
```

Comment:
It looks like the blurb entry used the PR number (gh-102502) rather than the issue number (gh-102493).